examples/sinter_example.py

```diff
@@ -22,24 +22,31 @@ def generate_example_tasks():
                 json_metadata={
                     'p': p,
                     'd': d,
-                    'rounds': d,
+                    'rounds': 3 * d,
                 },
             )
 
 
 def main():
     samples = sinter.collect(
         num_workers=10,
```

Comment:
Is there a way of setting this to the `number_of_available_cores - 1`.
The `-1` is there to ensure there is at least one core left for general purpose computing whilst the sims are running